src/vs/platform/terminal/common/xterm/shellIntegrationAddon.ts

```diff
@@ -48,7 +48,15 @@ const enum ShellIntegrationOscPs {
 	/**
 	 * Sequences pioneered by iTerm.
 	 */
-	ITerm = 1337
+	ITerm = 1337,
+	/**
+	 * Sequences by Cmder.
+	 */
+	Cmder = 9,
+	/**
+	 * Sequences by unknown terminal (maybe a command, rather than an indentifier).
+	 */
+	Unknown7 = 7
```

Comment:
These are actually just extensions to xterm, see https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h3-Operating-System-Commands how it goes to `Ps 6` and then there is a gap before `Ps 10`. I'd call them specific to what they do as they aren't a group of sequences like the other 3. How about this?

```
SetCwd = 7,
SetWindowsFriendlyCwd = 9 // We can always improve the name later if this isn't accurate
```